bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -193,3 +212,9 @@ message GetBookieInfoResponse {
 
 message StartTLSResponse {
 }
+
+message SyncResponse {
+    required StatusCode status = 1;
+    required int64 ledgerId = 2;
```

Comment:
Again, do we need these? If the bookie has acknowledged an unsynced entry, it means when that bookie gets the sync message, the entry will be synced.

Reply:
@ivankelly  Sorry, I can't understand your comment here